integration_tests/pom.xml

```diff
@@ -78,6 +78,42 @@
     </dependencies>
     <build>
         <plugins>
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>exec-maven-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>clean-previously-populated-config-file</id>
+                        <phase>package</phase>
```

Comment:
We need to make sure that this runs before the package phase such as generate-resources otherwise. It is hard to control order of executions across plugins in the same phase.

Reply:
I have removed this as you suggested.

Reply:
I meant the actual json generation, probably misplaced my comment here